lib/concurrent-ruby/concurrent/utility/processor_counter.rb

```diff
@@ -107,4 +143,16 @@ def self.processor_count
   def self.physical_processor_count
     processor_counter.physical_processor_count
   end
+
+  # Number of processors usable by the OS and used for process scheduling.
+  # In most cases it is the same than `processor_count`, however if the
+  # process is inside a cgroup with a dedicated CPU quota (typically Docker),
+  # the quota will be returned instead.
+  # For performance reasons the calculated value will be memoized on the first
+  # call.
+  #
+  # @return [Float] number of usable processors seen by the OS or Java runtime
+  def self.usable_processor_count
+    processor_counter.usable_processor_count
+  end
```

Comment:
Could you expose `cpu_quota` on `Concurrent` too and document it?

Reply:
Sure, no problem.